README.md

```diff
@@ -53,6 +67,8 @@ This will place three binaries in your `$GOBIN`;
 
 Make sure that your `$GOBIN` is in your `$PATH`.
 
+
+
```

Comment:
Remove this please, markdown squishes this all to one line anyway.

Reply:
roger.